tests/data/group_system/group_auditing/group_auditd_configure_rules/group_audit_privileged_commands/rule_audit_rules_privileged_commands_sudo/rhel7_augenrules_4294967295_configured.pass.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# profiles = xccdf_org.ssgproject.content_profile_ospp
+# remediation = bash
+
+echo "-a always,exit -F path=/usr/bin/sudo -F perm=x -F auid>=1000 -F auid!=4294967295 -k privileged" >> /etc/audit/rules.d/privileged.rules
+# This is a trick to fail setup of this test in rhel6 systems
+ls /usr/lib/systemd/system/auditd.service
```

Comment:
Do we want this in `*.pass.sh` test?

Reply:
Failure to setup scenario is reported in different way than rest of the execution/test. So yes, we do.